### PR TITLE
[RFC][UN-8746] Add error options

### PR DESCRIPTION
**Pull request by asantos00**
https://uniplaces.atlassian.net/browse/UN-8746

### CODE REVIEW

File: addon/styles/partials/_uni-dropdown.scss [hunk 1]
```diff
@@ -17,6 +17,10 @@
     }
   }
 
+  .uni-dropdown__button--error {
+    border-color: color('feedback', 'error');
```
Comment (fredcerdeira) [inline addon/styles/partials/_uni-dropdown.scss:21]: Double quotes here

File: addon/styles/partials/_uni-textarea.scss [hunk 2]
```diff
@@ -8,6 +8,12 @@
     resize: none;
   }
 
+  &--error {
+    textarea {
+      border-color: color('feedback', 'error');
```
Comment (fredcerdeira) [inline addon/styles/partials/_uni-textarea.scss:13]: Double quotes here

File: addon/styles/partials/_uni-textarea.scss [hunk 3]
```diff
@@ -8,6 +8,12 @@
...
+    }
+  }
+
   &-counter {
     @include body();
     color: color('text', 'subheader');
```
Comment (fredcerdeira) [inline addon/styles/partials/_uni-textarea.scss:19]: Double quotes here

**Review by fredcerdeira — CHANGES_REQUESTED** [review]

File: addon/styles/partials/_uni-dropdown.scss (hunk 1 above)
Comment (asantos00) [reply addon/styles/partials/_uni-dropdown.scss:21]: From what I've searched, we normally use single quotes.

https://github.com/uniplaces/ember-cli-uniq/search?p=2&q=color%28%22&type=&utf8=%E2%9C%93

**Review by mmelo — APPROVED** [review]
nothing to add to what @fredcerdeira identified. 👍

File: addon/styles/partials/_uni-dropdown.scss (hunk 1 above)
Comment (fredcerdeira) [reply addon/styles/partials/_uni-dropdown.scss:21]: I the lastest discussions about this topic we agreed that for CSS we should keep double quotes but confirm with another reviewer

**Comment by AndreJoaquim** [comment]
I believe that in this addon we use single quotes everywhere. Why did we do it? Can't remember. But it's better to keep a "bad" standard but consistent instead of creating unnecessary inconsistencies

**Review by AndreJoaquim — APPROVED** [review]

**Review by fredcerdeira — APPROVED** [review]

**Pull request merged**